### PR TITLE
feat: Add image serving

### PR DESCRIPTION
**Pull request by raphael0202**
We need to allow users to fetch images they uploaded.

Two other fixes are included in the PR.

### CODE REVIEW

File: app/schemas.py [hunk 1]
```diff
@@ -33,12 +33,24 @@ class ProductCreate(BaseModel):
 
 class ProductBase(BaseModel):
```
Comment (raphodn) [inline app/schemas.py:34]: I made a miss here, we could inherit from ProductCreate like the other schemas

Comment (raphodn) [reply app/schemas.py:34]: maybe do this in a seperate PR (inherit + pydantic description/examples)

Comment (raphodn) [reply app/schemas.py:34]: done here : https://github.com/openfoodfacts/open-prices/pull/58
merge 58 first then rebase ? :innocent:

Comment (raphael0202) [reply app/schemas.py:34]: I'm not in favor of multiplying PRs at this stage of the project, as it's more time-consuming. We can do rebase when merging the PR to have clean git histories.

Comment (raphael0202) [reply app/schemas.py]: Ok  I didn't understand you wanted to create it yourself ^^". I updated the PR to only keep serving images

Comment (raphodn) [reply app/schemas.py]: PRs might be slightly time consuming, but they help when multiple people are working on the project at the same time, avoiding conflitcts, and clarifying what changes were made :)

**Pull request merged**